Crane/buildSrc/src/main/java/com/example/crane/buildsrc/Dependencies.kt

```diff
@@ -57,8 +57,8 @@ object Libs {
         const val appcompat = "androidx.appcompat:appcompat:1.4.1"
 
         object Compose {
-            const val snapshot = ""
-            const val version = "1.2.0-rc01"
+            const val snapshot = "8738550"
```

Comment:
revert this and all of the other snapshot versions to ""

Reply:
Done in [6c132fc](https://github.com/android/compose-samples/pull/853/commits/6c132fc2f8783f344bdc25ef399207504f972aef). I also updated the upgrade script to clear the snapshot version when it is blank.